dataRetriever/resolvers/topicResolverSender/topicResolverSender.go

```diff
@@ -125,13 +143,15 @@ func (trs *topicResolverSender) SendOnRequestTopic(rd *dataRetriever.RequestData
 	fullHistoryPeers := make([]core.PeerID, 0)
 	if trs.currentNetworkEpochProviderHandler.EpochIsActiveInNetwork(rd.Epoch) {
 		crossPeers = trs.peerListCreator.PeerList()
```

Comment:
Can you rename on peerListCreator.PeerList() to peerListCreator.CrossShardPeerList()

Reply:
renamed